app/javascript/alchemy_admin/base.js

```diff
@@ -0,0 +1,75 @@
+/**
```

Comment:
I think it makes sense to make modules out of these functions and import them where we actually need them.